cassandra/cluster.py

```diff
@@ -139,7 +139,7 @@ def _is_gevent_monkey_patched():
     try:
         from cassandra.io.libevreactor import LibevConnection as DefaultConnection  # NOQA
     except ImportError:
-        from cassandra.io.asyncorereactor import AsyncoreConnection as DefaultConnection  # NOQA
+        from cassandra.io.asyncioreactor import AsyncioConnection as DefaultConnection  # NOQA
```

Comment:
We could also think about removing asyncore in a follow up PR.

Reply:
There's no rush on that,

And we can sync with upstream about when todo it, i.e. not to break current users that might have hardcore it